README.md

```diff
@@ -1,2 +1,18 @@
 # InnerCircle.EmailSender
 Service for sending emails via smtp
+
```

Comment:
I think we should add the section about SendGrid, for example:  
## Send Grid
In this service we use "SendGrid" for sending emails to recipients.  
You must have an account with API key that you will use in this app.  
If you don't have an account, you can create it [here](https://sendgrid.com/pricing/)  

***Notice***: make sure that you don't push your SendGrid credentials in this repo. Otherwise, SendGrid can block your account